### PR TITLE
Update README for bindata updates

### PR DESCRIPTION
**Pull request by philoserf**
The bindata.go file generated by bindata is part of package main. To move it into package cmd requires a line edit.

### CODE REVIEW

**Comment by philoserf** [comment]
This bullet list is actually sequential. Should it be a numbered list?

File: README.md [hunk 1]
```diff
@@ -280,6 +280,7 @@ follow the steps below:
 
 * Run `go-bindata data/`
```
Comment (philoserf) [inline README.md]: These are actually sequential steps. Should It be converted to a numbered list?

Comment (philoserf) [inline README.md]: This can be added to the Makefile in a manor that simplifies the readme and burden on the maintainers.

Comment (alejandroEsc) [reply README.md]: Open a PR with that and let's test it out!

**Comment by philoserf** [comment]
opened #204

**Pull request merged**